qemu/tests/timerdevice_host_time_back.py

```diff
@@ -0,0 +1,74 @@
+import re
+import time
+import logging
+
+from avocado.utils import process
+
+from virttest import error_context
+from virttest.compat_52lts import decode_to_text
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Check if guest hang/stall if host clock is set back
+
+    1) sync host system time with ntp server
+    2) boot guest with "-rtc base=utc, clock=host, driftfix=slew"
+    3) set host system time back
+    4) reboot guest and do some operation.
+
+    :param test: QEMU test object.
+    :param params: Dictionary with test parameters.
+    :param env: Dictionary with the test environment.
+    """
+    ntp_server = params["ntp_server"]
+    error_context.context("Sync host system time with ntpserver", logging.info)
+    process.system("ntpdate %s" % ntp_server)
+
+    vm = env.get_vm(params["main_vm"])
+    vm.verify_alive()
+    timeout = int(params.get("login_timeout", 360))
+    session = vm.wait_for_login(timeout=timeout)
```

Comment:
`wait_for_login()` actually covers `verify_alive()`, so it should be okay to drop #L30